class System_NotFiniteNumberException {
    constructor() {
        // System.Collections.IDictionary Data {get;}
        this.Data = undefined;

        // string HelpLink {get;set;}
        this.HelpLink = undefined;

        // int HResult {get;set;}
        this.HResult = undefined;

        // System.Exception InnerException {get;}
        this.InnerException = undefined;

        // string Message {get;}
        this.Message = undefined;

        // double OffendingNumber {get;}
        this.OffendingNumber = undefined;

        // string Source {get;set;}
        this.Source = undefined;

        // string StackTrace {get;}
        this.StackTrace = undefined;

        // System.Reflection.MethodBase TargetSite {get;}
        this.TargetSite = undefined;

    }

    // bool Equals(System.Object obj), bool _Exception.Equals(System.Object obj)
    Equals() {

    }

    // System.Exception GetBaseException(), System.Exception _Exception.GetBaseException()
    GetBaseException() {

    }

    // int GetHashCode(), int _Exception.GetHashCode()
    GetHashCode() {

    }

    // void GetObjectData(System.Runtime.Serialization.SerializationInfo info, System.Runtime.S...
    GetObjectData() {

    }

    // type GetType(), type _Exception.GetType()
    GetType() {

    }

    // string ToString(), string _Exception.ToString()
    ToString() {

    }

}

System_NotFiniteNumberException = new ProxyGenerator(System_NotFiniteNumberException);
